app/tests/spec/views/settings/gravatar_permissions.js

```diff
@@ -0,0 +1,128 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'chai',
+  'jquery',
+  'sinon',
+  'lib/promise',
+  'views/permissions',
+  'lib/metrics',
+  'lib/fxa-client',
+  'models/user',
+  '../../mocks/window',
+  '../../mocks/router',
+  '../../lib/helpers'
+],
+function (chai, $, sinon, p, View, Metrics,
+      User, WindowMock, RouterMock, TestHelpers) {
```

Comment:
I think there's something funny here. The `amdcheck` task said that `../../lib/helpers` was unused, but I definitely see `TestHelpers.{createEmail,isEventLogged}` used below.

It looks like we define 11 libraries, but only have 10 function args (with `lib/fxa-client` being the weird outlier in the middle).